src/app/components/shared-components.module.ts

```diff
@@ -0,0 +1,21 @@
+import { FlexLayoutModule } from '@angular/flex-layout';
+import { MatProgressSpinnerModule } from '@angular/material/progress-spinner';
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { CenteredLoadingSpinnerComponent } from './centered-loading-spinner/centered-loading-spinner.component';
+import { FooterComponent } from './footer/footer.component';
+
+
+
+@NgModule({
+  declarations: [CenteredLoadingSpinnerComponent],
```

Comment:
There shouldn't be a nested module here. this should go inside components folder inside the shared module

Reply:
I will refactor my modules